spec/spec_helper.rb

```diff
@@ -35,8 +35,41 @@ def inside_of_folder(folder)
   end
 end
 
+# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
-  config.expect_with(:rspec) { |c| c.syntax = :should }
   config.include SpecHelpers::InstanceMethods
   config.extend SpecHelpers::ClassMethods
+
+  config.filter_run_when_matching :focus
+  config.example_status_persistence_file_path = "spec/examples.txt"
+  config.warnings = true
+  config.order = :random
+  Kernel.srand config.seed
+
+  config.expect_with :rspec do |expectations|
```

Comment:
maybe a `# Remove this section when on rspec 4` ... or even detect rspec version and raise when on 4 ... seems like a bit overkill to me, we might as well take care of any issue once we are on rspec 4